src/SixLabors.Fonts/TextLayout.cs

```diff
@@ -665,7 +653,10 @@ private static void SubstituteBidiMirrors(FontMetrics fontMetrics, GlyphSubstitu
                     }
 
                     // Do not start a line following a break with breaking whitespace
```

Comment:
This fixes an issue discovered downstream where we were incorrectly trimming whitespace from the beginning of new lines following a mandatory line break.